Sources/Risk/RiskProviderError.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2022 Payoneer Germany GmbH
+// https://www.payoneer.com
+//
+// This file is open source and available under the MIT license.
+// See the LICENSE file for more information.
+
+import Foundation
+
+private let errorStringMaximumLength = 2000
```

Comment:
Is it an OPG requirement? If yes, please add a comment about it.

Reply:
It's defined in the ticket. @siebeprojects is there any documentation I can refer to here?

Reply:
The requirement is is that we should not send an error message longer than 2000 characters. I do not believe we need to refer to any documentation about that from code.

Reply:
I still think we should provide a link because in the future we will forget about this ticket and it will look like a "magic number".

```suggestion
// Defined in https://optile.atlassian.net/browse/PCX-2999
private let errorStringMaximumLength = 2000
```